__tests__/lib/mdast/index.test.ts

```diff
@@ -1,21 +1,43 @@
 import { mdast } from '../../../lib';
-import { readdir, readFile } from 'node:fs/promises';
+
+import nullAttributesMdx from './null-attributes/in.mdx?raw';
+import nullAttributesJson from './null-attributes/out.json';
+
+import tablesMdx from './tables/in.mdx?raw';
+import tablesJson from './tables/out.json';
+
+import variablesMdx from './variables/in.mdx?raw';
+import variablesJson from './variables/out.json';
+
+import inlineImagesMdx from './images/inline/in.mdx?raw';
+import inlineImagesJson from './images/inline/out.json';
+
+import inlineImageAttributesMdx from './images/inline-attributes/in.mdx?raw';
+import inlineImageAttributesJson from './images/inline-attributes/out.json';
```

Comment:
This is just a nicer experience, you got automatic re-runs, easier to add `.only`.